promtail/imageroot/state/promtail.yaml

```diff
@@ -0,0 +1,17 @@
+server:
+  http_listen_port: 9080
+
+clients:
+  - url: ${LOKI_URL}
```

Comment:
Who is actually replacing the placeholder?

Reply:
promtal; https://grafana.com/docs/loki/latest/clients/promtail/configuration/#use-environment-variables-in-the-configuration